.gitignore

```diff
@@ -74,4 +74,6 @@ server.auth.json
 private-key.json
 \.*.swp
 
-package-lock.json
\ No newline at end of file
+package-lock.json
+src/.prettierrc
```

Comment:
Can you perhaps rather configure this in your global .gitingore instead of the project? See https://gist.github.com/subfuzion/db7f57fff2fb6998a16c.